docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+ASM1 Property Package
+=====================
+
+This package implements property relationships for the treatment of wastewater using an activated sludge biological reactor as provided in `Henze, M. et al. (1987) <https://belinra.inrae.fr/doc_num.php?explnum_id=4467>`_.
+
+This Activated Sludge Model no.1 (ASM1) property package:
+   * supports only 'H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', and 'S_ALK' as components
+   * supports only liquid phase
+   * is formulated on a mass basis
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', 'S_ALK']"
+  "Phases", ":math:`p`", "['Liq']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Total volumetric flowrate", ":math:`Q`", "flow_vol", "None", ":math:`\text{m}^3\text{/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+   "Component mass concentrations", ":math:`C_j`", "conc_mass_comp", "[p]", ":math:`\text{kg/}\text{m}^3`"
+   "Alkalinity in molar concentration", ":math:`A`", "alkalinity", "[p]", ":math:`\text{kmol HCO}_{3}^{-}\text{/m}^{3}`"
+
+Parameters
```

Comment:
```suggestion
Stoichiometric Parameters
```
Although we generally have a section for just "Parameters", renaming this might be more in line with what's in the model as well as typical literature on ASM1.